Assets/Scripts/FollowPlayer.cs

```diff
@@ -13,7 +15,32 @@ void Start()
 
     void Update()
     {
+        if (_isSlowedFor <= 0)
+        {
+            _isSlowedFor = 0;
+            _slow = 0;
+        }
+        else
```

Comment:
else is unneeded 
just always count down _isSlowedFor

Reply:
no, because always counting down leads to a high number and hitting it with a slow with += timespan would do nothing, hitting with = timespan is in my opinion not wanted. Thus else is needed!

Reply:
I don't think SlowedFor time should stack

I think we should always count don't slowedFor and set it to timespan on hit

Reply:
group decision is needed